### DISCUSSION TITLE
Adding Mvc to Elsa workflow

### DISCUSSION THREAD

**Discussion by MohammedNash** [elsa-workflows/elsa-core#2520] [Q&A]
Can anyone help me to add mvc to elsa workflow. @sfmskywalker

**Comment by ghost** [comment]
Elsa designer is in Blazor if you want to do customization to designer and engine is in asp.net core api (already mvc)

what do you want to achieve, can you elaborate but more, please

**Comment by MohammedNash** [comment]
I want to create a web page where it contains textarea and approval/reject
Button in the form ,once i click either approval or reject  i want to pass
that text(in textarea)as a feedback through an email.
I have created a workflow using designer and not code approach.so i want to
achieve adding a new api where it contains forms.

**Comment by ghost** [comment]
no issues, create a HttpEndPoint (HttpEndPoint activity) which will receive
the data (textarea in the form) and have it saved in the variable using
SetVariable Activity and then send email activity and within the email body
use the variable to get the data

**Comment by MohammedNash** [comment]
Have you tried with any poc? if yes ,could you please share the same?

**Comment by MohammedNash** [comment]
Actually this where i got stucked as i am not able to get the data,may be
because iam not using the correct vatiable to get the data.
Can you please provide with variable to get the data from the form

**Reply by sfmskywalker** [reply]
Perhaps you could share your project? Then we can more easily see what you have tried and offer some suggestions.
